src/views/product-downloads-view/components/featured-tutorials-section/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { ReactElement } from 'react'
+import { IconPlay16 } from '@hashicorp/flight-icons/svg-react/play-16'
+import { useCurrentProduct } from 'contexts'
+import getTruncatedText from 'lib/get-truncated-text'
+import { FeaturedTutorial } from 'views/product-downloads-view/types'
+import CardLink from 'components/card-link'
+import Heading from 'components/heading'
+import ProductIcon from 'components/product-icon'
+import Text from 'components/text'
+import s from './featured-tutorials-section.module.css'
+
+interface FeaturedTutorialsSectionProps {
+  featuredTutorials: FeaturedTutorial[]
+}
+
+const FeaturedTutorialsSection = ({
+  featuredTutorials,
+}: FeaturedTutorialsSectionProps): ReactElement => {
+  const currentProduct = useCurrentProduct()
+
+  return (
+    <>
+      <Heading
+        className={s.sectionHeading}
+        level={2}
+        size={300}
+        slug="featured-tutorials"
+        weight="bold"
+      >
+        Featured Tutorials
+      </Heading>
+      <div className={s.cardGrid}>
+        {featuredTutorials.map(({ title, description, href }) => {
+          return (
+            <CardLink className={s.card} key={href} href={href}>
+              <div>
+                <Text className={s.cardTitle} size={300} weight="semibold">
+                  {title}
+                </Text>
+                <Text className={s.cardBody} size={200} weight="regular">
+                  {getTruncatedText(description, 80)}
+                </Text>
+              </div>
+              <div className={s.cardIcons}>
+                <IconPlay16 />
```

Comment:
I think the icons here denote whether or not the tutorial has a video or interactive example, maybe we can add a TODO to add support for that?
![Screen Shot 2022-02-25 at 10 52 26 AM](https://user-images.githubusercontent.com/1289701/155754984-52c0c4ea-b347-4a14-9b50-680b2b42cf1a.png)

Reply:
I'm pretty sure these will eventually be `LearnTutorialCard` components, so I'll add a TODO referencing that Asana ticket. 😊